plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/WorkspacesListItem.tsx

```diff
@@ -226,6 +230,37 @@ export const WorkspacesListItem = ({
   );
 };
 
+const deletingStatuses: readonly WorkspaceStatus[] = ['deleting', 'deleted'];
```

Comment:
Maybe we can make this `errorStatuses` or add a separate `errorStatuses`, to make sure `failed` is not considered to be online.